website/static/css/pages/contributor-page.css

```diff
@@ -89,6 +97,12 @@ th.remove {
   .btn-group-justified-vertical > .btn-group .btn {
     width: 100%;
   }
+  .remove-or-reject {
+    padding-top: 4px;
+    color: #343333;
```

Comment:
Oh sorry this was probably my fault. Preprints uses `#343333` because all of our apps are special snowflakes. OSF uses `#333333`.

<img width="671" alt="screen shot 2018-04-04 at 4 26 45 pm" src="https://user-images.githubusercontent.com/7131985/38332877-07712caa-3825-11e8-99c1-e43923e35d0b.png">